docs/grafana-ui.rst

```diff
@@ -0,0 +1,37 @@
+Grafana-UI
+==========
+
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using a operationnal cluster.
+
+  On your host, you will need to install kubectl.
+
+
+On your host, connect to the vagran VM
```

Comment:
Same remark as for #1182 - this documentation isn't targetting Vagrant deployments. We may mention it in some situations, but it shouldn't be as obvious as done here.

Reply:
If I may, here's a suggestion for replacing this procedure:

```
You will first need ``kubectl`` installed on your host.

To authenticate with the cluster, first retrieve the admin kubeconfig on your host:

.. code-block:: shell

   $ scp root@bootstrap:/etc/kubernetes/admin.conf ./admin.conf

Using this file, activate port-forwarding for Grafana on your host:

.. code-block:: shell

   $ kubectl -n monitoring --kubeconfig ./admin.conf port-forward svc/grafana 3000

Now, you can navigate to ``http://localhost:3000`` on your host, which should display Grafana UI.
```

Reply:
correction applied.